dtmi/azure/phoneasiotdevice/sensors/location-1.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:azure:PhoneAsIoTDevice:Sensors:Location;1",
+    "@type": "Interface",
+    "displayName": {
+      "en": "Location"
+    },
+    "contents": [
+      {
+        "@type": "Property",
+        "displayName": {
+          "en": "Enabled"
+        },
+        "name": "enabled",
+        "schema": "boolean",
+        "writable": true
+      },
+      {
+        "@type": "Telemetry",
+        "displayName": {
+          "en": "Location"
+        },
+        "name": "geolocation",
+        "schema": "dtmi:azure:PhoneAsIoTDevice:Sensors:Location:position;1"
```

Comment:
can we use geopoint instead of a custom schema? I doubt it will be recognized by Central